File: src/main/java/com/temelyan/pomoapp/repository/dataJpa/CrudTaskRepository.java
package com.temelyan.pomoapp.repository.dataJpa;

import com.temelyan.pomoapp.model.Task;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface CrudTaskRepository extends JpaRepository<Task, Integer> {
    List<Task> getAllByProjectId(int projectId);
}

File: src/main/java/com/temelyan/pomoapp/repository/dataJpa/CrudTaskRepository.kt
package com.temelyan.pomoapp.repository.dataJpa

import com.temelyan.pomoapp.model.Task
import org.springframework.data.jpa.repository.JpaRepository

interface CrudTaskRepository : JpaRepository<Task, Int> {
    fun getAllByProjectId(projectId: Int): List<Task>
}
